src/app/page.module.scss

```diff
@@ -3,7 +3,7 @@
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  padding: 6rem;
+  padding: 30px;
```

Comment:
It's better to use rem

Reply:
For what reason? I've never used them...